tests/data/wire/goal_state.xml

```diff
@@ -12,15 +12,15 @@
      <ContainerId>c6d5526c-5ac2-4200-b6e2-56f2b70c5ab2</ContainerId>
      <RoleInstanceList>
        <RoleInstance>
-         <InstanceId>MachineRole_IN_0</InstanceId>
+         <InstanceId>b61f93d0-e1ed-40b2-b067-22c243233448.MachineRole_IN_0</InstanceId>
```

Comment:
From these tests it seems that the might not always be present in the RoleConfigName. Is this guid necessary for putting logs in the host? Is it always guaranteed to be there? Any idea?

Reply:
The tests didn't contain this parameter simply because we never knew about it / needed it until now. Real world goal states come with this parameter.